Array in Javascript/README.md

```diff
@@ -76,6 +76,54 @@ The 4 element would be pushed to the end of the array. Notice that push returns
 Full code can be found at [here](src/insertion.js)
 ### Micro-Challenge
 Can you find A way to clone an array. Create a PR and add it here as Micro-Challenge Solution
```

Comment:
You need to remove the below statement.
`Create a PR and add it here as Micro-Challenge Solution.`